backend/server.js

```diff
@@ -20,7 +31,36 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Hi there Allan!')
+})
+
+// Sign up endpoint
+app.post('/signup', async (req, res) => {
+  try {
+    const { name, email, password } = req.body
+    const user = await new User({ name, email, password: bcrypt.hashSync(password) })
+    user.save()
+    res.status(201).json({ id: user._id, accessToken: user.accessToken })
+  } catch (err) {
+    res.status(400).json({ message: 'Could not create user!', errors: err.errors })
+  }
+})
+
+// Login endpoint
+app.post('/login', async (req, res) => {
+  const user = await User.findOne({ email: req.body.email })
```

Comment:
To be consistent you could destructure this as in the `/users` endpoint:
```
const { email, password } = req.body
const user = await User.findOne({ email })
```

Reply:
We are only using the email from the req.body. It feels unnecessary to destructure it.